services/app-api/utils/testing/fixtures/completionStatus/mlr-template.test.json

```diff
@@ -189,39 +189,74 @@
+                },
+                {
+                  "type": "externalLink",
+                  "content": "\"Managed Care Enrollment by Program and Plan\"",
+                  "props": {
+                    "href": "https://www.medicaid.gov/medicaid/managed-care/enrollment-report/index.html",
+                    "target": "_blank",
+                    "aria-label": "Link opens in new tab"
+                  }
+                }
+              ]
+            }
+          },
           {
             "id": "report_programName",
             "type": "textarea",
             "validation": "text",
             "props": {
-              "label": "I. Program name",
-              "hint": "Enter the name of the program(s) for this MLR report. A program is defined by a specified set of benefits and eligibility criteria that are articulated in a contract between the state and managed care plans. States providing plan-level MLR results by program should enter the individual program name in the field below. States providing plan-level MLR results across programs should list all relevant program names in the field below, separated by commas."
+              "label": "J. Program name",
+              "hint": "Enter the name of the program(s) for this MLR report. A program is defined by a specified set of benefits and eligibility criteria that are articulated in a contract between the state and managed care plans. States providing plan-level MLR results by program should enter the individual program name in the field below. States providing plan-level MLR results across programs should list all relevant program names in the field below, separated by commas."
             }
           },
           {
             "id": "report_programType",
             "type": "checkbox",
             "validation": "checkbox",
             "props": {
-              "label": "J. Program type",
-              "hint": "Select the program type definition(s) that best describes the program(s) entered in “I. Program name”.",
+              "label": "K. Program type",
+              "hint": "Select the program type definition(s) that best describes the program(s) entered in “J. Program name”.",
               "choices": [
+                {
+                  "id": "G4inXZJWYFsDZYGk75cW7L",
+                  "label": "Comprehensive MCO"
+                },
+                {
+                  "id": "6krUQZyhpmHHoagQWgUzxx",
+                  "label": "PIHP"
+                },
+                {
+                  "id": "VNp9mDXgv9ixL2H8JHVBBE",
+                  "label": "PAHP"
+                },
                 {
                   "id": "cQhGnyBmiXszLqrx6b2Cp8",
-                  "label": "Behavioral Health Only"
+                  "label": "Behavioral Health"
                 },
                 {
-                  "id": "G4inXZJWYFsDZYGk75cW7L",
+                  "id": "J8PmLy8nKxTK55XJch2jDw",
                   "label": "Comprehensive MCO"
```

Comment:
Question: Why are there two identical options for "Comprehensive MCO"?

Reply:
Ben, that is a fantastic question I'm glad you asked. Its the flatRoutes output that get generated upon report creation. Because this is a test fixture thats using a created report, it has this in here.